core/server/master/src/test/java/alluxio/master/file/PermissionCheckerTest.java

```diff
@@ -175,7 +175,7 @@ public static void beforeClass() throws Exception {
         .setOwner(TEST_USER_1.getUser()).setGroup(TEST_USER_1.getGroup());
     sNestedFileContext = CreateFileContext
         .defaults(CreateFilePOptions.newBuilder().setBlockSizeBytes(Constants.KB)
-            .setMode(TEST_NORMAL_MODE.toShort()))
+            .setMode(TEST_NORMAL_MODE.toShort()).setRecursive(true))
```

Comment:
why is this change needed? did the default change

Reply:
I had mistakenly removed it before. It should be recursive.